test/commands/lightning/dev/component.nut.ts

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
are these test files necessary? they don't do anything

Reply:
Brian will add these tests. Just created the initial file structure